{"exception":{"stacktrace":"org.apache.dubbo.remoting.RemotingException: client(url: dubbo:\/\/169.254.98.54:20880\/com.overstar.order.export.api.IOrderService?anyhost=true&application=overstar-open&bean.name=ServiceBean:com.overstar.order.export.api.IOrderService&category=providers&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&heartbeat=60000&interface=com.overstar.order.export.api.IOrderService&lazy=false&methods=cancel,cancelBySso,cancelFailed,confirmFinishBySso,deleteBatch,cancelFinished,cancelByQuartz,confirmFinish,getBySn,getById,create,confirmFinishByQuartz,notifyPayed,orderCountBySso,sendShip&path=com.overstar.order.export.api.IOrderService&pid=10632&protocol=dubbo&qos.enable=false&register=true&register.ip=172.25.206.49&release=2.7.3&remote.application=test-order&revision=0.0.1-SNAPSHOT&side=consumer&sticky=false&timestamp=1574950631256) failed to connect to server \/169.254.98.54:20880, error message is:Network is unreachable: no further information: \/169.254.98.54:20880\n\tat org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)\n\tat org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)\n\tat org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)\n\tat org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)\n\tat org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)\n\tat org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)\n\tat org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)\n\tat org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)\n\tat org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)\n\tat java.lang.Thread.run(Thread.java:748)\nCaused by: io.netty.channel.AbstractChannel$AnnotatedSocketException: Network is unreachable: no further information: \/169.254.98.54:20880\n\tat sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)\n\tat sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)\n\tat io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)\n\tat io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:670)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:617)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:534)\n\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906)\n\tat io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\n\tat io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)\n\t... 1 more\nCaused by: java.net.SocketException: Network is unreachable: no further information\n\t... 12 more","exception_class":"org.apache.dubbo.remoting.RemotingException","exception_message":"client(url: dubbo:\/\/169.254.98.54:20880\/com.overstar.order.export.api.IOrderService?anyhost=true&application=overstar-open&bean.name=ServiceBean:com.overstar.order.export.api.IOrderService&category=providers&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&heartbeat=60000&interface=com.overstar.order.export.api.IOrderService&lazy=false&methods=cancel,cancelBySso,cancelFailed,confirmFinishBySso,deleteBatch,cancelFinished,cancelByQuartz,confirmFinish,getBySn,getById,create,confirmFinishByQuartz,notifyPayed,orderCountBySso,sendShip&path=com.overstar.order.export.api.IOrderService&pid=10632&protocol=dubbo&qos.enable=false&register=true&register.ip=172.25.206.49&release=2.7.3&remote.application=test-order&revision=0.0.1-SNAPSHOT&side=consumer&sticky=false&timestamp=1574950631256) failed to connect to server \/169.254.98.54:20880, error message is:Network is unreachable: no further information: \/169.254.98.54:20880"},"source_host":"PV-X00253891","method":"doTask","level":"ERROR","message":" [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [172.25.206.49:0 -> \/169.254.98.54:20880]], dubbo version: 2.7.3, current host: 172.25.206.49","mdc":{},"@timestamp":"2019-11-28T14:18:14.879Z","file":"ReconnectTimerTask.java","application":"service_product_new","line_number":"51","thread_name":"dubbo-client-idleCheck-thread-1","@version":1,"logger_name":"org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask","class":"org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask"}
